tests/unit_tests/rigid_pendulum_poc/test_math_utilities.cpp

```diff
@@ -0,0 +1,79 @@
+#include <gtest/gtest.h>
+
+#include "src/rigid_pendulum_poc/utilities.h"
+
+namespace openturbine::rigid_pendulum::tests {
+
+TEST(MathUtilitiesTest, CloseTo) {
+    ASSERT_TRUE(close_to(1., 1.));
+    ASSERT_TRUE(close_to(1., 1. + 1e-7));
+    ASSERT_TRUE(close_to(1., 1. - 1e-7));
+    ASSERT_FALSE(close_to(1., 1. + 1e-5));
+    ASSERT_FALSE(close_to(1., 1. - 1e-5));
```

Comment:
It's good to test close_to with values around the tolerance.
Perhaps it would be more clear to use the actual tolerance constant, with `kTol*10` and `kTol/10`.
Will tolerance always be 1e-6? Could some problems benefit from a smaller or larger tolerance? Should it be related to edge size in the mesh, or some other problem characteristic? If it should just always be 1e-6 then these tests are fine as is.

Reply:
> It's good to test close_to with values around the tolerance.
> Perhaps it would be more clear to use the actual tolerance constant, with `kTol*10` and `kTol/10`.

Yup, I agree. Thanks for the suggestion.

> Will tolerance always be 1e-6? Could some problems benefit from a smaller or larger tolerance? Should it be related to edge size in the mesh, or some other problem characteristic? If it should just always be 1e-6 then these tests are fine as is.

Just want to point out the tolerance is not static, only the default is 1e-6. We should be able to change the value based on the problem at hand like you suggested above.

https://github.com/Exawind/openturbine/blob/a3d068c26222fa1c75c249a38de4577acab9af8e/src/rigid_pendulum_poc/utilities.h#L16